src/Tablebot/Plugins/Reminder.hs

```diff
@@ -119,12 +119,16 @@ reminderCron = do
                         "Reminder to <@" ++ show uid ++ ">! " ++ content
                     P.delete (entityKey r)
 
+reminderHelp :: HelpPage
+reminderHelp = HelpPage "remind" "ask the bot to remind you to do things in the future" "**Reminders**\nSend a reminder to yourself or others. Pick a date and time, and the tablebot will poke you to remember at your preordained moment.\n\n*Usage:* `remind \"reminder\" at <time>`" [] 
```

Comment:
Please multiline long strings. Since Esquelito already uses TemplateHaskell, you can use [Text.RawString.QQ](https://hackage.haskell.org/package/raw-strings-qq-1.1/docs/Text-RawString-QQ.html).